docroot/themes/custom/mass_theme/overrides/js/inline-overlay.js

```diff
@@ -0,0 +1,34 @@
+(function ($) {
+  'use strict';
+
+  var tocFocusableElements = $('.js-inline-overlay').find(':focusable');
+  var firstFocusableElement = tocFocusableElements[0];
+  var lastFocusableElement = tocFocusableElements[tocFocusableElements.length - 1];
+  var tocButtons = $('.ma__toc--hierarchy__container .js-accordion-link');
+  var tocLastButton = tocButtons[tocButtons.length - 1];
+
+  $(tocLastButton).keydown(function (e) {
+    var key = (e.keyCode ? e.keyCode : e.witch);
+    var lastContainer = $(tocLastButton).closest('.ma__toc--hierarchy__accordion');
```

Comment:
```suggestion
    var lastContainer = $(tocLastButton).closest('.ma__toc--hierarchy__accordion');
```
can you move this out of the keydown function

Reply:
The change is made as suggested.